### ISSUE TITLE
Add activation ops to TTNN

### ISSUE THREAD

**Issue by KalaivaniMCW** [tenstorrent-metal/tt-metal#5169]
**Is your feature request related to a problem? Please describe.**
We need a new operation to perform hyperbolic operations and inverse hyperbolic operations similar to what was implemented in torch.
ex. [Heaviside](https://pytorch.org/docs/stable/generated/torch.heaviside.html#torch.heavisidel) , [Relu6](https://pytorch.org/docs/stable/generated/torch.nn.ReLU6.html)
Following are the ops needed
relu6, elu, hardshrink, hardswish, hardtanh, heaviside, leaky_relu, log_sigmoid, mish, prelu, relu_max, relu_min, sigmoid, sign, softshrink, softsign, swish, clip

**Describe the solution you'd like**
This will apply activation function on the given input tensor and return one output tensor.

**Describe alternatives you've considered**
It's possible that ttnn tensor could be converted to a Pytorch tensor and then apply activation functions with Pytorch function. The result would then be returned using ttnn.from_torch operation.

**Additional context**
The reference implementation of an activation function
```
import torch

tensor = torch.tensor([-1.5, 0, 2.0])
values = torch.tensor([0.5])

result_tensor = torch.heaviside(tensor, values)
print(result_tensor)
```

**Comment by KalaivaniMCW** [comment]
merged to main #5217
